### ISSUE TITLE
[BUG] document dont save from draft to publish

### ISSUE THREAD

**Issue by SZEINER** [Leantime/leantime#1929]
![obrazek](https://github.com/Leantime/leantime/assets/77299927/04c45727-151c-4d27-aaa0-e404819d4c62)

**What set up are you using**
* [ ] Self Hosted

**Describe the bug**
If i create document under "wiki" ant want to safe from draft to publich. There is no action and again "draft" and dont save other content there.

**Leantime Version**
Leantime 2.4 beta 5

**Server**
Apache

**PHP / MySQL Version**
PHP 8.1

**Comment by jiriks74** [comment]
Duplicate of #1920 

If you don't want to wait for the updated Docker image run this in your container:
```bash
docker exec -it <container_id> sh -c "sed -i 's/public function updateArticle(Article \$Article)/public function updateArticle(Article \$article)/g' /var/www/html/app/Domain/Wiki/Repositori
es/Wiki.php"
```

Please close this as it's resolved

**Comment by marcelfolaron** [comment]
just released 2.4-beta-6 which should fix this
